src/app/docs/docs.component.spec.ts

```diff
@@ -32,18 +32,18 @@ describe('DocsComponent', () => {
     .compileComponents();
   }));
 
-  beforeEach(() => {
-    fixture = TestBed.createComponent(DocsComponent);
-    component = fixture.componentInstance;
-    fixture.detectChanges();
-  });
+  // beforeEach(() => {
```

Comment:
Add a comment about why this code is commented or better yet, but it behind a descriptive predicate 
Bare uncommented code is easily broken/deleted during cleanup